### PR TITLE
[Tizen] Prepare executorch build for Tizen

### PR DESCRIPTION
**Pull request by anyj0527**
- Add building executorch filter subplugin package in spec file.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [ ]Passed [ ]Failed [X]Skipped

### CODE REVIEW

**Review by niley7464 — APPROVED** [review]
LGTM 👍

**Review by jaeyun-jung — APPROVED** [review]

File: packaging/nnstreamer.spec [hunk 1]
```diff
@@ -37,6 +37,8 @@
 %define		snpe_support 1
 %define		trix_engine_support 1
 %define		onnxruntime_support 1
+# Enable executorch_support when executorch package is ready.
+%define		executorch_support 0
```
Comment (songgot) [inline packaging/nnstreamer.spec:41]: ```executorch_llama_support``` is right?

File: packaging/nnstreamer.spec [hunk 2]
```diff
@@ -885,6 +904,13 @@ NNStreamer's datareposrc/sink plugins for reading and writing files in MLOps Dat
 %define enable_onnxruntime -Donnxruntime-support=disabled
 %endif
 
+# Support executorch
+%if 0%{?executorch_support}
+%define enable_executorch -Dexecutorch-support=enabled
+%else
+%define enable_executorch -Dexecutorch-support=disabled
```
Comment (myungjoo) [inline packaging/nnstreamer.spec:911]: Make executorch-llama-support explicit or add a comment that executorch-llama-support will be enabled alongside.

**Review by myungjoo — APPROVED** [review]

File: packaging/nnstreamer.spec (hunk 1 above)
Comment (myungjoo) [reply packaging/nnstreamer.spec:41]: executorch_support is right.

**Pull request merged**